server.js

```diff
@@ -20,8 +21,61 @@ app.use(cors());
 app.use(express.json());
 
 // Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+app.get('/', (req, res) => {
+  res.send('Welcome to my book API!');
```

Comment:
I love being welcomed to a page, but I also have to give you a tip about running `npm i express-list-endpoints` and then using that on the first page. Then you get a list of all possible endpoints in the code and makes it possible to just click on them to see the different lists 😄